src/features/zaaksysteem/service.ts

```diff
@@ -9,6 +9,27 @@ import {
 } from "@/services";
 import type { Zaak } from "./types";
 import type { ZaakDetails } from "./types";
+import type { ContactmomentViewModel } from "../contactmoment";
+
+type Roltype = "behandelaar" | "initiator";
+
+const getNamePerRoltype = (zaak: any, roltype: Roltype): string => {
+  const behandelaar = zaak.embedded.rollen.find(
+    (rol: any) =>
+      rol.betrokkeneType === "medewerker" &&
+      rol.omschrijvingGeneriek === roltype
+  );
+
+  const identificatie = behandelaar?.embedded?.betrokkeneIdentificatie;
+
+  if (!identificatie) return "Onbekend";
+
+  const voornaam = identificatie.voornamen ?? "";
+  const tussenvoegsel = identificatie.voorvoegselGeslachtsnaam ?? "";
+  const achternaam = identificatie.geslachtsnaam ?? "";
+
+  return `${voornaam} ${tussenvoegsel} ${achternaam}`;
```

Comment:
nitpicking, but if there's no tussenvoegsel there would be two spaces

Reply:
Nice catch, fixed.